doc/osh.1

```diff
@@ -5,45 +5,47 @@
 .Nm osh
 .Nd A bash-compatible UNIX shell
 .Sh SYNOPSIS
-.Nm 
+.Nm
 .Op Ar options
 .Ar script
 .Op Ar args
-.Br
-.Nm 
+.Nm
 .Op Ar options
 .Fl c
 .Op command_string
 .Op args
 .Sh DESCRIPTION
-OSH is a command-line shell. It behaves like busybox and `osh` is a symlink to it.
+.Nm OSH
+is a command-line shell.
+It behaves like busybox and `osh` is a symlink to it.
 .Sh OPTIONS
 OSH accepts POSIX \fBsh\fR flags, with the following differences:
 .Bl -tag -width Ds
 .It Fl n
-only validate the syntax.  Also prints the AST.
+only validate the syntax.
+Also prints the AST.
 .It Fl Fl show-ast
 print the AST in addition to executing.
 .It Fl Fl ast-format
 what format the AST should be in
 .El
-.Sh AUTHOR
+.Sh SEE ALSO
+Developer documentation for
+.Nm OSH
+may be found online at
+.Lk http://www.oilshell.org/release/0.6.pre22/doc/osh-quick-ref.html
+.Sh AUTHORS
 Written by
 .An Andy Chu .
 .Sh REPORTING BUGS
 Please report bugs to
 .Aq Lk https://github.com/oilshell/oil/issues
 .Sh COPYRIGHT
```

Comment:
Don't put license info in the manpage, it's not needed. You could stick it up in the top of the file as a series of comments like:

```mdoc
.\" Copyright © 2016 Andy Chu. All rights reserved.
.\"
.\" Licensed under the Apache License, Version 2.0 (the "License");
.\" You may not use this file except in compliance with the License.
.\"
.\" You may obtain a copy of the License at:
.\" http://www.apache.org/licenses/LICENSE-2.0
```